### PR TITLE
JSDK-1281 Local Video Filter example.

### PR DESCRIPTION
**Pull request by manjeshbhargav**
@markandrus ,

This is the PR for the Local Video Filter example. I also alphabetized the examples in this PR.
Let me know what you think.

My next PR will be about refactoring based on your suggestions.

### CODE REVIEW

**Comment by markandrus** [comment]
@manjeshbhargav can you include screenshots? will take a look

File: package.json [hunk 1]
```diff
@@ -4,16 +4,18 @@
...
     "build:examples:localvideosnapshot": "copyfiles -f examples/localvideosnapshot/src/helpers.js examples/localvideosnapshot/public && browserify examples/localvideosnapshot/src/index.js > examples/localvideosnapshot/public/index.js",
+    "build:examples:mediadevices": "copyfiles -f examples/mediadevices/src/helpers.js examples/mediadevices/public && browserify examples/mediadevices/src/index.js > examples/mediadevices/public/index.js",
     "build:quickstart": "browserify quickstart/src/index.js > quickstart/public/index.js",
+    "clean": "npm-run-all clean:*",
+    "clean:examples": "npm-run-all clean:examples:*",
+    "clean:examples:localvideofilter": "rimraf examples/localvideofilter/public/index.js examples/localvideofilter/public/helpers.js",
+    "clean:examples:localvideosnapshot": "rimraf examples/localvideosnapshot/public/index.js examples/localvideosnapshot/public/helpers.js",
+    "clean:examples:mediadevices": "rimraf examples/mediadevices/public/index.js examples/mediadevices/public/helpers.js",
```
Comment (markandrus) [inline package.json]: We should work to generalize this. Listing everything here won't scale well. We could add a `scripts/build-examples.js`, for example.

**Comment by manjeshbhargav** [comment]
@markandrus Here's the screenshot

screen shot 2017-06-07 at 1 48 14 pm

File: package.json (hunk 1 above)
Comment (manjeshbhargav) [inline package.json]: @markandrus I'll include this as part of the refactoring PR that will include the changes you suggested in the other examples.

File: examples/localvideofilter/public/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            <pre class="language-javascript"></pre>
+          </div>
+        </div>
+      </div>
+      <div class="col-sm-4">
+        <div class="card">
+          <div class="card-block">
+            <h4 class="card-title">Local Video (raw)</h4>
+            <video id="videoinputpreview" autoplay></video>
+            <select id="filter">
```
Comment (markandrus) [inline examples/localvideofilter/public/index.html]: I think you should include `class="form-control"` on the `<select>`. You may also need to wrap the component in a `<div class="form-group">` to get the correct styling.

**Pull request merged**